### PR TITLE
Misc Edits

### PR DESCRIPTION
**Pull request by sappenin**
* Update normative and informative references
* Update feedback
* Clarify introduction, definition, and examples
* Separate examples from definition
* Remove superfluous examples/explanations

Signed-off-by: sappenin <sappenin@gmail.com>

### CODE REVIEW

**Comment by sappenin** [comment]
The `txt`, `html` and `xml` files are all auto-genereated, so to review this PR, plz look at the `.md` files.

File: src/spec/payid-uri.md [hunk 1]
```diff
@@ -50,242 +63,202 @@ informative:
...
-   `payid:alice$www.example.com`.
+   In addition, a 'payid' is case-insensitive and therefore should be
+   normalized to lowercase.  For example, the URI 
+   "PAYID:aLICE$www.EXAMPLE.com" is equivalent to 
+   "payid:alice$www.example.com".
+   
+   Note that both the 'acctpart' and 'host' components of a 'payid' may
+   contain one or more dollar-sign characters. However, because a 'host' 
+   SHOULD also be a valid DNS domain, that portion of a 'payid' will
+   generally not include a dollar-sign. Therefore, applications should
```
Comment (aanchal4) [inline src/spec/payid-uri.md:135]: should --> SHOULD?

File: src/spec/payid-uri.md [hunk 2]
```diff
@@ -50,242 +63,202 @@ informative:
...
+   scheme via the following construction: 
+   
+     'payid:apollo$wallet.example.com'
+
+   One possible PayID scenario is for an account to be registered with a 
+   payment service provider using an identifier that is associated with some 
+   other service provider. For example, a user with the email address 
+   "alice@example.net" might register with a wallet website whose domain
+   name is "wallet.example.com". In order to facilitate payments to/from
+   Alice, the wallet service provider might offer alice a PayID using alice's
```
Comment (aanchal4) [inline src/spec/payid-uri.md:151]: alice and Alice are are used interchangeably. It should be `alice` for the payid and `Alice` to refer to the person Alice?

File: src/spec/payid-uri.md (hunk 1 above)
Comment (sappenin) [reply src/spec/payid-uri.md]: Fixed.

File: src/spec/payid-uri.md (hunk 2 above)
Comment (sappenin) [reply src/spec/payid-uri.md]: Fixed.

**Pull request merged**